### ISSUE TITLE
[Feature Request] media_player as button on cardGrid/cardEntites (turn_on/turn_off)

### ISSUE THREAD

**Issue by joBr99** [joBr99/nspanel-lovelace-ui#121]

**Comment by hatran0884** [comment]
Yes, i am thinking the same, a cardentities with 3 first entities will be scripts of playlist playing and the last entity will be a media player having 5 buttons : volume up, volume down, play, next songs and previous song.. it’d be super convenient without going to a next page to control the player. Hope you are thinking the same :D
